src/Groups/directproducts.jl

```diff
@@ -361,15 +361,17 @@ julia> C = cyclic_group(2)
 Pc group of order 2
 
 julia> A = automorphism_group(Q)
-Aut( <pc group of size 8 with 3 generators> )
+Automorphism group of
+  pc group of order 8
 
 julia> au = A(hom(Q,Q,[Q[1],Q[2]],[Q[1]^3,Q[2]^3]))
 [ x, y ] -> [ x*y2, y*y2 ]
 
 julia> f = hom(C,A,[C[1]],[au])
 Group homomorphism
   from pc group of order 2
-  to aut( <pc group of size 8 with 3 generators> )
+  to automorphism group of
+    pc group of order 8
```

Comment:
Hmmm... probably should distinguish between "one line" and "detailed" printing mode here... perhaps it should be
```suggestion
  to automorphism group of pc group of order 8
```
or in fact just
```suggestion
  to Aut( pc group of order 8 )
```
or so ... ?

Reply:
So some options here would include
```suggestion
  to automorphism group of pc group
```
or perhaps really this should be using that the group has been assigned to a variable `C` ...
```suggestion
  to automorphism group of C
```
or
```suggestion
  to Aut( C )
```
or for that matter, since the automorphism group has been assigned to variable `A`, it could be
```suggestion
  to A
```